sum/contracts/src/deploy.ts

```diff
@@ -0,0 +1,57 @@
+import * as dotenv from 'dotenv';
```

Comment:
it is not possible to use the sc-deployer package here ?

Reply:
we use it, imported line 5.